configure/toolchain-i686.cmake

```diff
@@ -0,0 +1,15 @@
+set(CMAKE_CROSSCOMPILING TRUE)
```

Comment:
IMO this file should be merged as part of #792 or at least as a separate commit in this PR, just to keep the history clear.

Reply:
@mliszcz Yeah #792 would be a good fit as well, but as I wanted it to be ready for tomorrow I snuck it in here. I've now extracted it into its own commit. Thanks for reminding me.